docs/how-to/cluster-cfg.rst

```diff
@@ -18,44 +18,43 @@ The microceph cluster configuration CLI supports setting, getting, resetting and
 
   .. code-block:: shell
 
-    $ sudo microceph cluster config set cluster_network 10.5.2.165/16
+    $ sudo microceph cluster config set cluster_network 10.5.0.0/16
 
 2. Config value for a particular key could be queried using the 'get' command:
 
   .. code-block:: shell
 
     $ sudo microceph cluster config get cluster_network
-    +---+-----------------+---------------+
-    | # |       KEY       |     VALUE     |
-    +---+-----------------+---------------+
-    | 0 | cluster_network | 10.5.2.165/16 |
-    +---+-----------------+---------------+
+    +---+-----------------+-------------+
+    | # |       KEY       |     VALUE   |
+    +---+-----------------+-------------+
+    | 0 | cluster_network | 10.5.0.0/16 |
+    +---+-----------------+-------------+
 
 3. A list of all the configured keys can be fetched using the 'list' command:
 
   .. code-block:: shell
 
-    $ sudo microceph cluster config set public_network 10.5.2.165/16
+    $ sudo microceph cluster config set public_network 10.5.0.0/16
     $ sudo microceph cluster config list
-    +---+-----------------+---------------+
-    | # |       KEY       |     VALUE     |
-    +---+-----------------+---------------+
-    | 0 | cluster_network | 10.5.2.165/16 |
-    +---+-----------------+---------------+
-    | 1 | public_network  | 10.5.2.165/16 |
-    +---+-----------------+---------------+
+    +---+-----------------+-------------+
+    | # |       KEY       |     VALUE   |
+    +---+-----------------+-------------+
+    | 0 | cluster_network | 10.5.0.0/16 |
+    +---+-----------------+-------------+
+    | 1 | public_network  | 10.5.0.0/16 |
+    +---+-----------------+-------------+
 
 4. Resetting a config key (i.e. setting the key to its default value) can performed using the 'reset' command:
 
   .. code-block:: shell
 
+   $ sudo microceph cluster config reset cluster_network
```

Comment:
Is there any specific reason for cleaning both cluster/public network config ? The original intention was to suggest that these 2 configs are independent and need not be set/reset together.

Reply:
When I read the doc in the first run, I interpreted it as setting "10.5.2.165/16" as the custom network then resetting it. I was confused for one minutes finding the default value from the command output as in "setting the key to its default value". Because the only visible value in the output is still "10.5.2.165/16".

I thought it would be a good idea to show the empty table to be clean that the default value means no entry in the Ceph config database.

Reply:
Aaah, that makes sense. Thanks for the explanation.